material/problems/VectorGeometry/PolygonNormal/PolygonNormal.tex

```diff
@@ -1,65 +1,125 @@
 %
-%
 \begin{minipage}[t]{0.68\columnwidth}
-In many engineering applications (such as stress and aerodynamics computations using finite elements, as well as CAD and graphical display), objects are represented using a group of polygons. Being able to compute a ``normal'' vector perpendicular to each polygon is important for all these applications.
+  \minipageTopAnchor
+  In many engineering applications -- such as stress and aerodynamics
+  computations using finite elements, as well as CAD and graphical display --
+  objects are represented using a group of polygons (usally triangles)
+  collectively called a ``mesh". Being able to compute a ``normal'' vector
+  perpendicular to each polygon is important for all these applications.
 \end{minipage}
+%
 \hfill
+%
 \begin{minipage}[t]{0.25\columnwidth}
-\flushright
-\vspace*{0pc}
-%\includegraphics[width=0.95\columnwidth]{dolphin.jpg}
-%\includegraphics[width=0.9\columnwidth]{wing_mesh.png}
-\includegraphics[width=0.97\columnwidth]{wing_robot.jpg}
+  \minipageTopAnchor
+  \flushright
+  \includegraphics[width=0.9\columnwidth]{wing_robot.jpg}
 \end{minipage}
-%\\[0.0pc]
-%For example, for drawing a polygon on screen, it is necessary to compute the illumination in a way that simulates realistic lighting. A simple model for diffuse illumination at each vertex (corner) of a polygon is defined as $I_D = I_L * \uvecWithHat{n} \cdot \uvecWithHat{L}$, where:
-%\begin{itemize}
-%\item $I_L$ is the (scalar) strength of the incoming light.
-%\item $\uvecWithHat{L}$ is the unit vector from the vertex to the light source.
-%\item $\uvecWithHat{n}$ is the ``unit-normal" for the polygon (that is, the unit vector perpendicular to the polygon).
-%\end{itemize}
 %
-\\[0.5pc]
-In the figure below, the \uvecxyz{a} ``coordinate" measures of each vertex from $A_o$ are labeled.
+\\[0.45pc]
+In the figure below, the \uvecxyz{a} measures of the position vector to each
+vertex from $A_o$ are labeled.
 \\[0.0pc]
-For example, $\posvec{A_o}{P_1} \equals[\;] 3~\uvecx{a} \plus[\;] 2~\uvecy{a} \plus[\;] 0~\uvecz{a}$.
+For example, $\posvec{A_o}{P_1} \equals[\;] 3~\uvecx{a} \plus[\;] 2~\uvecy{a}
+\plus[\;] 0~\uvecz{a}$.
+%
+\begin{center}
+  \includegraphicsAB[width=7cm]{polygon-solution.png}{polygon_normal.png}
+\end{center}
+%
 \begin{enumerate}
-\item \underline{Compute} a vector \bvec{n} that is perpendicular to the polygon, expressed in \uvecxyz{a}.
-\item \underline{Explain} in detail how to get \uvecWithHat{n}, the unit vector in the same direction as \bvec{n}. \underline{Sketch} \uvecWithHat{n} on the figure.
-\item \underline{Explain} why there are actually two possible \uvecWithHat{n} vectors for the polygon.
-%\item Compute \posvec{P_0}{Q}, and hence compute the unit vector $\uvecWithHat{L}$.
-%\item For a light source of strength $I_L = 0.5$, compute the diffuse illumination $I_D$.
+  \item \underline{Compute} a vector \bvec{n} that is perpendicular to the polygon,
+  expressed in terms of \uvecxyz{a}.
```

Comment:
Hi Adam,

Please excuse me for the delays.
For the last couple of days I have been constantly traveling and taking
care of some personal issues.

I have seen your comments and took care of them. I have a questions
regarding the code that is quoted. The line is already indented 2 spaces
after the begin{enumerate}. Would you like it to be indented two lines
more/meaning that the things following it will be also indented/ or do you
want the whole enumerate block to start with two spaces identation?

BR,
Alex
On Jul 20, 2015 2:25 AM, "Adam Leeper" <notifications@github.com> wrote:

> In material/problems/VectorGeometry/PolygonNormal/PolygonNormal.tex
> <https://github.com/aleeper/EngineeringMechanicsShared/pull/5#discussion_r34964429>
> :
>
> >  \begin{enumerate}
> > -\item \underline{Compute} a vector \bvec{n} that is perpendicular to the polygon, expressed in \uvecxyz{a}.
> > -\item \underline{Explain} in detail how to get \uvecWithHat{n}, the unit vector in the same direction as \bvec{n}. \underline{Sketch} \uvecWithHat{n} on the figure.
> > -\item \underline{Explain} why there are actually two possible \uvecWithHat{n} vectors for the polygon.
> > -%\item Compute \posvec{P_0}{Q}, and hence compute the unit vector $\uvecWithHat{L}$.
> > -%\item For a light source of strength $I_L = 0.5$, compute the diffuse illumination $I_D$.
> > +  \item \underline{Compute} a vector \bvec{n} that is perpendicular to the polygon,
> > +  expressed in terms of \uvecxyz{a}.
>
> indent this line 2 spaces
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aleeper/EngineeringMechanicsShared/pull/5/files#r34964429>
> .
>